DatadogInternal/Sources/Models/SessionReplay/SessionReplayConfiguration.swift

```diff
@@ -20,6 +20,16 @@ public enum SessionReplayPrivacyLevel: String {
     case maskUserInput = "mask_user_input"
 }
 
+/// Available privacy levels for image masking in the Session Replay.
+public enum SessionReplayImagePrivacyLevel {
```

Comment:
question/ Why we're adding this configuration in `DatadogInternal` not `DatadogSessionReplay` 🤔? It made sense for `privacyLevel` that is made for `DatadogWebViewTracking` (so: type sharing use case). But the `ImagePrivacyLevel` which has nothing to do with WVT should be IMO located in:
- [`DatadogSessionReplay/Sources/SessionReplayConfiguration.swift`](https://github.com/DataDog/dd-sdk-ios/blob/develop/DatadogSessionReplay/Sources/SessionReplayConfiguration.swift), no?

Reply:
// cc @maxep who may have more context on SR config in `DatadogInternal`.

Reply:
Good catch.

So there're two angles here. First - maybe we want to pass image privacy to the webview - in that case it's in the correct place. Otherwise - you're right, it shouldn't be here.

I'll double check with @maxep

Reply:
Yes, `SessionReplayImagePrivacyLevel` was moved to `DatadogInternal` because it is shared with `DatadogWebViewTracking`. Browser SDK does not support image privacy setting from the bridge, and I don't think there is such feature in Browser anyway. So let's keep it out of `DatadogInternal` for now 👍